avocado/core/dispatcher.py

```diff
@@ -14,6 +14,7 @@
 
 """Extensions/plugins dispatchers."""
 
+import logging
```

Comment:
How about using the `configure` dispatcher and allow it to report keywords or values (dict). One of the supported arguments would be `claimed_stdout:True|False`? Avocado would make a note of that and in the future we'd be able to add other args if needed.